components/forms/cct/CctCalcCreate.tsx

```diff
@@ -111,352 +110,372 @@ export function CctCalcCreate() {
+                                                          .wte,
+                                                      label: `${
+                                                        (values.changes[index]
+                                                          .wte as number) * 100
+                                                      }%`
+                                                    }
+                                                  : null
+                                              }
+                                            />
+                                          )}
+                                          {!(
+                                            errors.changes as CctCalculationErrors["changes"]
+                                          )?.[index]?.wte &&
                                             (values.changes[index]
-                                              .wte as number) *
-                                              100
-                                        ) && (
-                                          <span data-cy="wte-custom-warn">
-                                            <FieldWarningMsg warningMsg="Actioning a custom WTE change might not be possible; it will require Dean approval." />
-                                          </span>
-                                        )}
-                                    </Col>
-                                  </Row>
-                                </div>
-                              </Fragment>
-                            ))}
-                          </div>
-                        )}
-                      />
-                    </>
-                  ) : null}
-                  <br />
-                  {values.programmeMembership.id && isValid && dirty && (
-                    <Row>
-                      <Col width="full">
-                        <Button
-                          data-cy="cct-calc-btn"
-                          onClick={() => handleSubmit}
-                        >
-                          Calculate new completion date
-                        </Button>
-                      </Col>
-                    </Row>
-                  )}
-                </Container>
-              </Form>
-            )}
-          </Formik>
-          {name && created && lastModified && (
-            <CalcDetails
-              created={created}
-              lastModified={lastModified}
-              name={name}
-            />
+                                              ?.wte as number) >
+                                              (values.programmeMembership
+                                                .wte as number) && (
+                                              <span data-cy="wte-increase-return-warn">
+                                                <FieldWarningMsg warningMsg="Actioning an increase in WTE hours needs a suitable post to be available, which might not be possible." />
```

Comment:
Possible phrasing:

Increasing WTE hours will require a suitable post to be available. Be aware that this will need to be confirmed.